pkg/knode-manager/adapters/k8s/help.go

```diff
@@ -0,0 +1,97 @@
+package k8sadapter
+
```

Comment:
help.go or helps.go ? I did not see any code file named help.go

Reply:
chang name to util.go?